packages/front-end/pages/settings/index.tsx

```diff
@@ -481,112 +536,220 @@ const GeneralSettingsPage = (): React.ReactElement => {
                       <Field
-                        label="Cron String"
+                        label="Statistics Engine"
                         className="ml-2"
-                        disabled={hasFileConfig()}
-                        {...form.register("updateSchedule.cron")}
-                        placeholder="0 */6 * * *"
-                        onFocus={(e) => {
-                          updateCronString(e.target.value);
-                        }}
-                        onBlur={(e) => {
-                          updateCronString(e.target.value);
-                        }}
-                        helpText={<span className="ml-2">{cronString}</span>}
+                        options={[
+                          {
+                            display: "Bayesian",
+                            value: "bayesian",
+                          },
+                          {
+                            display: "Frequentist",
+                            value: "frequentist",
+                          },
+                        ]}
+                        {...form.register("statsEngine")}
                       />
                     </div>
-                  )}
+                  </div>
                 </div>
+                <div className="form-check mb-2 mt-2">
+                  <input
+                    type="checkbox"
+                    className="form-check-input"
+                    name="advanced-options"
+                    checked={showAdvancedOptions}
+                    onChange={(e) => {
+                      setShowAdvancedOptions(!!e.target?.checked);
+                    }}
+                    id="checkbox-advanced"
+                  />
 
-                <Field
-                  label="Statistics Engine"
-                  className="ml-2"
-                  options={[
-                    {
-                      display: "Bayesian",
-                      value: "bayesian",
-                    },
-                    {
-                      display: "Frequentist",
-                      value: "frequentist",
-                    },
-                  ]}
-                  {...form.register("statsEngine")}
-                />
+                  <label
+                    htmlFor="checkbox-advanced"
+                    className="form-check-label"
+                  >
+                    Show Advanced Options
+                  </label>
+                </div>
+                {showAdvancedOptions && (
+                  <div className="bg-light p-3 my-3 border rounded">
+                    <div>
+                      <h5 className="font-weight-bold mb-4">
+                        Advanced Options - use caution
+                      </h5>
+                    </div>
+                    <div>
+                      <div className="form-group mb-2 mr-2 form-inline flex-wrap">
+                        <Field
+                          label="Bayesian confidence threshold"
```

Comment:
Sorry, one more nit, can we say "Bayesian chance to win threshold" or "Bayesian significance threshold"?